packages/toolpad-app/src/toolpadDataSources/rest/client.tsx

```diff
@@ -530,7 +532,8 @@ function QueryEditor({
                 globalScope={globalScope}
                 globalScopeMeta={globalScopeMeta}
                 liveValue={paramsEditorLiveValue}
-                jsRuntime={jsBrowserRuntime}
+                jsRuntime={jsServerRuntime}
+                envVarNames={envVarNames}
```

Comment:
We can also pass the whole `env` to these props now instead of just the `envVarNames`, and rename all `envVarNames` to `env`.